### ISSUE TITLE
Python failed to initialize if there is no "PYTHONHOME" defined.

### ISSUE THREAD

**Issue by lichaozhou** [opentap/OpenTap.Python#41]
OpentTap.Python 3.0 RC

If "PYTHONHOME" is not defined,  it will crash when doing python initiallization.
from https://github.com/pythonnet/pythonnet/issues/1534, I tried to set PythonEngine.PythonHome, and this issue is resolved.

```
Runtime.PythonDLL = @"D:\WinPython\WPy64-3830\python-3.8.3.amd64\python38.dll";
PythonEngine.PythonHome = @"D:\WinPython\WPy64-3830\python-3.8.3.amd64";
PythonEngine.Initialize(false, false);
```

@rmadsen-ks

**Comment by rmadsen-ks** [comment]
Hi @lichaozhou,
Thanks for submitting an issue.

It is not possible to support you setting PythonHome directly, because by the time your plugin is loaded PythonEngine.Initialize has already been called. Instead, I suggest you set 

Python Library Path inside Settings. This is how it looks like for me:

![image](https://user-images.githubusercontent.com/24250062/191210168-b09c721c-83a8-4236-9940-bad7655ce38a.png)

This should work for you:

![image](https://user-images.githubusercontent.com/24250062/191210424-ae9e575c-42a0-4220-ad8d-5abb5619de10.png)

**Comment by rmadsen-ks** [comment]
Closing this now, please reopen if the issue persists.

**Comment by lichaozhou** [comment]
> Hi @lichaozhou, Thanks for submitting an issue.
> 
> It is not possible to support you setting PythonHome directly, because by the time your plugin is loaded PythonEngine.Initialize has already been called. Instead, I suggest you set
> 
Hi @rmadsen-ks 
The sample code is not a python plugin, I just use the code to duplicate the crash issue with pythonnet.
I mean in OpenTap.Python PythonInitializer.cs, it does not set PythonEngine.PythonHome to python path. so it will also crash if PYTHONHOME is not defined.
I am using a portable python,  it does dose not depend on any system evrionment variable.

**Comment by rmadsen-ks** [comment]
Sorry, I don't understand. Could you try to clarify a bit what you are trying to do?

**Comment by lichaozhou** [comment]
Hi @rmadsen-ks 
 
Sorry for the confused. Let me clarify my scenario.
1. OpenTAP Engine 9.18.5+0328e9b4 X64
2. OpenTap.Python  3.0.0-rc+b49e92cd
3.  WinPython (python 3.8.3 for x64)
4. Editor 9.17.4 for x64

 In python setting, I set the path to WinPython 
![image](https://user-images.githubusercontent.com/19421133/191250801-b5e6ce0c-2166-4098-bb67-96d127320909.png)

Then run the Editor.exe, it encountered crash.

After I define "PYTHONHOME" in system environment to WinPython,  it can load Python plugins successfully.
![image](https://user-images.githubusercontent.com/19421133/191251591-9077eba0-5089-4431-9aa6-b8f2c6aeedf7.png)
![image](https://user-images.githubusercontent.com/19421133/191251852-c6950f6d-34fc-4dc8-9812-2097b520cc31.png)

I was wondering if it is possible that no depending on the "PYTHONHOME" ENV variable,  then I made a change to the source code according to https://github.com/pythonnet/pythonnet/issues/1534, in PythonInitializer.cs PythonInitializer->InitInternal()

```
 Runtime.PythonDLL = pyLoc;
 PythonEngine.PythonHome = pyPath;  // chnage,  set python home
 PythonEngine.Initialize(false);
```
After this change,  python plugins also can be loaded successfully without "PYTHONHOME"
